src/services/geometry/richgeo/RichGeo.h

```diff
@@ -24,6 +24,7 @@ namespace richgeo {
       double            size;
       dd4hep::Position  surface_centroid;
       dd4hep::Direction surface_offset; // surface centroid = volume centroid + `surface_offset`
+      dd4hep::Direction surface_normZdir; //normal Z direction of surface
```

Comment:
Remove if you agree this is not needed (see other comment)
```suggestion
```

Reply:
Removed and surface_offset is now used.